### PR TITLE
UCM/TEST: Generate VM_UNMAP event for shmat(REMAP) and mmap(FIXED)

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by yosefe** [comment]
adding @bureddy

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7244/ for details (Mellanox internal link).

**Comment by bureddy** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7245/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7247/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7248/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7249/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@hoopoepg pls take a look

File: test/gtest/ucs/test_async.cc [hunk 1]
```diff
@@ -416,12 +416,12 @@ UCS_TEST_P(test_async, max_events, "ASYNC_MAX_EVENTS=4") {
 
 UCS_TEST_P(test_async, many_timers) {
 
-    for (int count = 0; count < 1000; ++count) {
+    for (int count = 0; count < 4010; ++count) {
```
Comment (hoopoepg) [inline test/gtest/ucs/test_async.cc]: magic numbers? :)

Comment (yosefe) [reply test/gtest/ucs/test_async.cc]: to reach the limit of id's with the previous (buggy) implementation

**Pull request merged**